.github/workflows/lint.yml

```diff
@@ -76,15 +76,26 @@ jobs:
           --tags-path aten/src/ATen/native/tags.yaml \
           --deprecated-functions-path "tools/autograd/deprecated.yaml"
 
-        RC=0
-        # Run lintrunner on all files
-        if ! lintrunner --force-color --all-files --tee-json=lint.json 2> /dev/null; then
+        # Run lintrunner on just the changed files (a fast validation)
+        lintrunner --force-color --tee-json=lint.json > /dev/null 2>&1
+        RC=$?
+        if [ $RC -ne 0 ]; then
           echo ""
           echo -e "\e[1m\e[36mYou can reproduce these results locally by using \`lintrunner\`. (If you don't get the same results, run \'lintrunner init\' to update your local linter)\e[0m"
           echo -e "\e[1m\e[36mSee https://github.com/pytorch/pytorch/wiki/lintrunner for setup instructions.\e[0m"
           RC=1
         fi
 
+        if [ $RC -eq 0 ]; then
+          # Run lintrunner on all files
+          if ! lintrunner --force-color --all-files --tee-json=lint.json 2> /dev/null; then
```

Comment:
I'm very confused about why this `if` statement works historically, since it seems to only show the error message if the lintrunner command _succeeded_